### DISCUSSION TITLE
Some Issues with the Latin

### DISCUSSION THREAD

**Discussion by rl1234567890** [DivinumOfficium/divinum-officium#3231] [General]
Forgive my Latin OCD, but I found these inconsistencies rather irritating:

1) omit. This appears when some part is not said. There is no such form in Latin. Maybe change to omittitur?
2) Omnia    Plures. I get the meaning: one can choose between printing the whole office ore some parts, but why one  is neuter (all things) and one feminine (more, in the sense of more hours, I surmise)?
3) Ordo Majus A. D.  2023 Totum. Ordo is masculine, not neuter. Also, I like the j, but it's not used in the

**Comment by APMarcello3** [comment]
1) "Omit." is merely used as an abbreviation for "omittitur" or "omittuntur", in case of singular or plural elements to be omitted.

2) Changed "Omnia" to "Omnes".

3) "Totum" changed to "Totus". The letter J is used in pre-1960 printings of the Office, so it will remain for this case.

Cf. commit bcdbeada8.
